Makefile

```
@@ -1,8 +1,21 @@
-IMAGE_NAME=quay.io/theforeman/pulpcore
-IMAGE_TAG=3.73.8
+PROJECT=pulp
```

I'd suggest to we stick to `$PROJECT-oci-images`. That means either rename the repo to `pulp-oci-images` or change this to
```suggestion
PROJECT=pulpcore
```